src/Platform/Platform.php

```diff
@@ -113,13 +160,36 @@ public function loggedIn()
     public function login($username = '', $extension = '', $password = '')
```

Comment:
You need to change signature to `login($options)` but still support old-style parameters.